File: Demais/PROGCOMP - Sequência - D.cpp
// PROGCOMP - Sequência - D

#include <iostream>
#include <iomanip>
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

float salario_base;
double salario_novo;
float gratificacao = 0.05;
float impostos = 0.07;

using namespace std;

int main() {

cin>>salario_base;

gratificacao = gratificacao*salario_base;
impostos = impostos*salario_base;

salario_novo = salario_base+gratificacao-impostos;

cout<<fixed<<setprecision(2)<<"novo: "<<salario_novo<<endl;
cout<<fixed<<setprecision(2)<<"gratificacao: "<<gratificacao<<endl;
cout<<fixed<<setprecision(2)<<"imposto: "<<impostos<<endl;

}

File: Demais/PROGCOMP - Sequência - D.py
#PROGCOMP - Sequência - D

salario_novo = float
gratificacao = float(0.05)
impostos = float(0.07)


salario_base = float (input())

gratificacao = gratificacao*salario_base
impostos = impostos*salario_base

salario_novo = (salario_base+gratificacao-impostos)

print("novo: %.2lf"%salario_novo)
print("gratificacao: %.2f"%gratificacao)
print("imposto: %.2f"%impostos)


